decidim-core/app/helpers/decidim/participatory_process_steps_helper.rb

```diff
@@ -13,5 +13,25 @@ def step_class(step, past)
       status = past ? "" : "timeline__item--inactive"
       step.active? ? "timeline__item--current" : status
     end
+
+    # Returns the formatted date for the given step
+    #
+    # step - the given ParticipatoryProcessStep
+    #
+    # Returns a String.
+    def formatted_date_for(step)
+      formatted_date = ""
+
+      if step.start_date
+        formatted_date += localize(step.start_date, format: "%d/%m/%Y")
```

Comment:
See these links for ideas of formats:

http://guides.rubyonrails.org/i18n.html#adding-date-time-formats 
https://github.com/svenfuchs/rails-i18n/blob/master/rails/locale/ca.yml